images/fcos-bastion-image/root/usr/bin/hpsol

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ "${#}" -lt 1 ]; then
+    echo "Usage: ${0} <host_id> <command>"
+    exit 1
+fi
+
+host_id="${1}"
+command="${2}"
```

Comment:
do we need "${command}" and `shift`?

I believe using `shift` and then `"${@}"` instead of `${command}` should be the safest way to ensure all the next paramters are passed over to the ssh session